AmazonChimeSDK/AmazonChimeSDK/internal/analytics/DefaultMeetingStatsCollector.swift

```diff
@@ -27,13 +29,20 @@ import Foundation
     }
 
     public func getMeetingStats() -> [AnyHashable: Any] {
+        
+        let meetingReconnectDurationMs = (meetingStartReconnectingTimeMs == 0
+                                          || meetingReconnectedTimeMs == 0
+                                          || meetingReconnectedTimeMs < meetingStartReconnectingTimeMs)
+        ? 0 : (meetingReconnectedTimeMs - meetingStartReconnectingTimeMs)
```

Comment:
Is there anywhere `meetingStartReconnectingTimeMs` and `meetingReconnectedTimeMs` need to be reset after this. Or it is handled by `resetMeetingStats`? The function seems always reset both. How it ensures not to happen when `meetingStartReconnectingTimeMs` in non-zero but `meetingReconnectedTimeMs` is.

Reply:
`updateMeetingStartReconnectingTimeMs()` will reset `meetingReconnectedTimeMs`, this func will be called when reconnecting starts.

> How it ensures not to happen when meetingStartReconnectingTimeMs in non-zero but meetingReconnectedTimeMs is.

I'm a bit confused, this is a valid state. When reconnecting starts, `meetingStartReconnectingTimeMs` will be non-zero, `meetingReconnectedTimeMs` will be zero.

Reply:
I guess the question is more about when `getMeetingStats` and `resetMeetingStats` are called. If they are called together, then `resetMeetingStats` may clear state of reconnection in progress (i.e., non-zero `meetingStartReconnectingTimeMs` and zero `meetingReconnectedTimeMs`). Otherwise, would the same reconnection report multiple reconnect duration?